src/components/ShareSocialCta/index.js

```diff
@@ -116,3 +116,7 @@ export default function(props) {
     </Location>
   );
 }
+
+function getLocation() {
+  return window.location.pathname;
```

Comment:
how come you've added this `getLocation()` function? it doesn't appear to be called anywhere.